app/experimenter/experiments/tests/test_models.py

```diff
@@ -1398,6 +1399,19 @@ def test_should_have_test_builds_true(self):
         experiment = ExperimentFactory(type=Experiment.TYPE_PREF)
         self.assertTrue(experiment.should_have_test_builds)
 
+    @parameterized.expand(
+        [
+            (False, Experiment.TYPE_ADDON),
+            (False, Experiment.TYPE_GENERIC),
+            (False, Experiment.TYPE_PREF),
+            (False, Experiment.TYPE_ROLLOUT),
+            (True, Experiment.TYPE_MESSAGE),
+        ]
+    )
+    def test_should_have_telemetry_event(self, expected, type):
+        experiment = ExperimentFactory(type=type)
+        self.assertEqual(experiment.should_have_telemetry_event, expected)
```

Comment:
I realized we should be testing all those boolean model methods like this, lets us test all the cases with a 'single' test.